test-requirements.txt

```diff
@@ -14,3 +14,4 @@ eventlet
 cython>=0.20,<0.28
 packaging
 asynctest; python_version > '3.4'
+pyOpenSSL
```

Comment:
We actually might be able to get away with `twisted[tls]` here, and maybe we should recommend that to users. (this knowledge earned yesterday while trying to get the duration tests to work :P)